src/backend/app.js

```diff
@@ -21,10 +22,86 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+// Respond with all meals in the future (relative to the when datetime)
+app.get("/future-meals", async (req, res) => {
+  try{
+    const today = new Date();
+    const futureMeals = await knex("Meal")
+      .where("_when", ">", today)
+      .select()
+      if(futureMeals.length === 0){
+        res.status(404).json({data: null , message : "No future meal available. chack later"});
+        return
+      }
+      res.status(200).json({data : futureMeals , message: "ok"});
+    }catch(err){
+      console.error(err)
+      res.status(500).json({data : null , message : "An error"});
```

Comment:
Error message could be more descriptive, like "An internal server error occurred", so that users of the server know that the error happens due to a bug on the server